packages/cloud-cognitive/src/components/Datagrid/Datagrid.stories.js

```diff
@@ -46,6 +46,10 @@ import mdx from './Datagrid.mdx';
 
 import styles from './_storybook-styles.scss';
 
+// import pkg from '../../settings';
+
+// const blockClass = `${pkg.prefix}--datagrid`;
```

Comment:
@faraznaseem019 

Let's clean this up. Unless the commented out is required for reason, it should be removed.

Though I'm almost certain there no longer needed as this code has been moved to lines 779-781